tests/lib/types/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "dom",
```

Comment:
A note that the `dom` library seems unnecessary as we probably don't want to deal with DOM stuff in this package.